test/parallel/test-vm-symbols.js

```diff
@@ -19,7 +19,9 @@ const context = new Document();
 vm.createContext(context);
 
 assert.strictEqual(context.getSymbolValue(), 'foo',
-                   'should return symbol-keyed value from the outside');
+                   `should return symbol-keyed value from the outside.
```

Comment:
It would be better if we just remove the assertion message (third arugment) here.